pom.xml

```diff
@@ -20,6 +20,17 @@
 
     <properties>
         <java.version>17</java.version>
+        <starter.data.jpa.version>2.7.5</starter.data.jpa.version>
```

Comment:
let's keep it like this, added tech task for the future `[tech] maven dependency and dependencyManagement`